docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -1,21 +1,22 @@
 == Installing Jenkins plugins
-Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be installed by including them in Ansible playbook or using the Jenkins UI.
-
 NOTE: Plugins for Jenkins can be found at https://plugins.jenkins.io
 
+Digger will by default install required Jenkins plugins. If more Jenkins plugins are required, they can be installed by listing them in a task in Ansible playbook or using Jenkins UI.
```

Comment:
Why do we swap the two sections above?

Reply:
wanted to have an external ref first, yeah makes sense to swap it back

Reply:
resolved